entries/n/netacad.com.json

```
@@ -0,0 +1,16 @@
+{
+  "Cisco Network Academy": {
+    "domain": "netacad.com",
```

Instead of having two identical images, it's better to just link to the other image :wink: 
```suggestion
    "domain": "netacad.com",
    "img": "cisco.com.svg",
```